ngraph/frontend/onnx_import/src/op/reduce.cpp

```diff
@@ -66,19 +126,36 @@ namespace ngraph
 
                 template <typename OpType>
                 std::shared_ptr<ngraph::Node>
-                    make_ng_reduction_op(const Node& node, const Output<ngraph::Node>& ng_input)
+                    make_ng_reduction_op(const Node& node,
+                                         const Output<ngraph::Node>& ng_input,
+                                         bool axes_as_attr = true)
                 {
-                    const auto reduction_axes = get_reduction_axes(node);
                     const std::int64_t keepdims =
                         node.get_attribute_value<std::int64_t>("keepdims", 1);
 
-                    const auto op_node = std::make_shared<OpType>(
-                        ng_input, reduction_axes, static_cast<bool>(keepdims));
-
-                    return op_node;
+                    const auto reduction_axes = axes_as_attr ? get_reduction_axes_from_attr(node)
+                                                             : get_reduction_axes_from_input(node);
+                    if (reduction_axes != nullptr)
+                    {
+                        return std::make_shared<OpType>(
+                            ng_input, reduction_axes, static_cast<bool>(keepdims));
+                    }
+                    else
+                    {
+                        return op::set_1::identity(node).at(0).get_node_shared_ptr();
```

Comment:
I see that `op::set_1::identity` wraps the input with `Add` op, is it necessary here?
Can we just return the input?

`return node.get_ng_inputs().at(0).get_node_shared_ptr();` 
or even 
`return  ng_input.get_node_shared_ptr();` 
(as we passed the input it as `Output<ngraph::Node>& ng_input` arg to this function)

```suggestion
                        return ng_input.get_node_shared_ptr();
```

Reply:
Test that covers that situation (onnx_model_reduce_sum_13_axes_empty_with_noop) passes after this change and I don't see any negative impact. Does anyone know if Kasia's proposed change has any negative effects?

@tomdol @tsocha @postrational